braidpool-primitives/src/bead/mod.rs

```diff
@@ -64,6 +69,62 @@ impl Bead {
         // TODO: Implement this function.
         unimplemented!()
     }
+
+    #[inline]
+    pub fn is_parent_of(&self, child_bead_hash: BeadHash) -> bool {
```

Comment:
How do you intend to use `is_*` functions? I personally dislike a bunch of helper functions like this, I'd rather write something like (pythonic) `if bead in otherbead.parents { ... }`. Unless this presents a problem with borrowing in Rust...